pkg/ruler/api.go

```diff
@@ -685,6 +685,12 @@ func (a *API) CreateRuleGroup(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
+	if err := a.ruler.AssertMinRuleEvaluationInterval(userID, time.Duration(rg.Interval)); err != nil {
+		level.Error(logger).Log("msg", "limit validation failure", "err", err.Error(), "user", userID)
```

Comment:
I don't think this should be logged as an error, this should be a warning at most: nothing failed, it's just a user who sent a wrong request.